src/routes/SheltersPege/SheltersPege.js

```diff
@@ -1,14 +1,20 @@
 import React, { Component } from 'react';
-import { isShelterAdmin, Section } from '../../components/Utils/Utils';
+import { isShelterAdmin, Section, Input } from '../../components/Utils/Utils';
 import './SheltersPege.css';
 import SheltersService from '../../services/sheltersService';
 import * as logUtils from '../../components/Utils/Logger';
 import PetList from '../../components/PetList';
 import PetsService from '../../services/petsService';
 import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import USStates from '../../components/Utils/USStates.json'
 var log = logUtils.getLogger()
 
-export default class SheltersPege extends Component {
+export default class SheltersPage extends Component {
```

Comment:
I think the folder and Files need to be renamed to say "SheltersPage" too. Same with the .css file